src/BattleCalculatorStatCrawler-Freewar/BattleCalculatorStatCrawler-Freewar.user.js

```diff
@@ -98,4 +108,94 @@ function routine() {
 }
 
 // Start the routine function
-routine();
\ No newline at end of file
+routine(); .listusersrow table tr td');
```

Comment:
sth broken here? missing starting `'`, see the syntax highlighting:

![pic](https://i.imgur.com/wsa17Cp.png)

Reply:
Wenn ich es richtig sehe, hast du gerade den commit gecheckt der die beiden scripts total gebroken hat, wie gesagt, bin kein github profi ^^

https://github.com/Zabuzard/BattleCalculator-Freewar/pull/4/commits/6503e0b3c0d3ef0c91bc0988659c5986f42ef38f
dieser letzte commit macht es rückgängig und dein request sollte da behoben sein :)

Reply:
oh, tatsache 😆